### PR TITLE
Update Snapshot to correct milestone

### PR DESCRIPTION
**Pull request by maiksensi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,13 +30,14 @@
 
     <properties>
         <!-- project version -->
-        <revision>2.0.3-SNAPSHOT</revision>
+        <revision>2.1.0-SNAPSHOT</revision>
```
Comment (WtfJoke) [inline pom.xml]: Only this line should change.

Comment (maiksensi) [reply pom.xml]: the file was not formatted properly. If you want I can create a separate MR, but the changes are in format only, so I actually count that as trivial and don't think this should be done in another MR.

Comment (maiksensi) [reply pom.xml]: as discussed lets make two commits.

**Comment by WtfJoke** [comment]
With 2 commits its great :) Thanks! 👍

**Pull request merged**